src/components/Clickable.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react'
+import clsx from 'clsx'
+
+export function Clickable(
+  props: React.ComponentProps<'div'> & {
+    disabled?: boolean
+  },
+) {
+  const ref = React.useRef<HTMLDivElement>(null)
+  return (
+    <div
+      {...props}
+      role="button"
+      tabIndex={props.disabled ? -1 : 0}
+      ref={ref}
+      onKeyDown={(event) => {
```

Comment:
Might remove the need to cast it to unknown
```suggestion
      onKeyDown={(event:React.MouseEvent<HTMLDivElement, MouseEvent>) => {
```

Reply:
No go!